consumer/guides/innstall_upm.md

```diff
@@ -0,0 +1,120 @@
+---

```

Comment:
Can you move this file under /consumer/guides/upm/ ? Also the file name should be install_upm.md